controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -163,40 +163,48 @@ void client_steering_task::steer_sta()
         /*
         * send disallow to all others
         */
-        agent_mac = database.get_node_parent_ire(hostap);
-        if (!cmdu_tx.create(0,
-                            ieee1905_1::eMessageType::CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE)) {
-            LOG(ERROR)
-                << "cmdu creation of type CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE, has failed";
-            return;
-        }
-
-        auto association_control_block_request_tlv =
-            cmdu_tx.addClass<wfa_map::tlvClientAssociationControlRequest>();
-        if (!association_control_block_request_tlv) {
-            LOG(ERROR) << "addClass wfa_map::tlvClientAssociationControlRequest failed";
-            return;
+        auto hostap_vap_lits = database.get_hostap_vap_list(hostap);
+        auto ssid            = database.get_hostap_ssid(target_bssid);
+        for (auto &hostap_vap : hostap_vap_lits) {
+            if (hostap_vap.second.ssid == ssid) {
```

Comment:
Remove extra indentation:
```suggestion
            if (hostap_vap.second.ssid != ssid) {
                continue;
            }
            .
            .
            .
```